requirements.txt

```diff
@@ -7,3 +7,5 @@ typing_extensions
 librosa
 pbr
 expelliarmus
+torch==1.12.0+cu113
+torchaudio==0.12.0+cu113
```

Comment:
cu113 means CUDA 11.3, so this is installing a fixed version of pytorch with CUDA backend